src/docker-compose.yaml

```diff
@@ -0,0 +1,107 @@
+version: "3.8"
```

Comment:
I think we can remove this line.

Regarding to https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element

Reply:
removed